flexibleengine/resource_flexibleengine_cce_cluster_v3.go

```diff
@@ -444,11 +444,32 @@ func resourceCCEClusterV3Update(d *schema.ResourceData, meta interface{}) error
 
 	if d.HasChange("description") {
 		updateOpts.Spec.Description = d.Get("description").(string)
```

Comment:
**updateOpts** is only used when `description` has changed, so suggest to put its definition in the if statement.

Reply:
sure, fixed